backend/src/packages/journal-entries/journal-entry.service.ts

```diff
@@ -83,8 +85,20 @@ class JournalEntryService implements Service {
     return item.toObject();
   }
 
-  public delete(): ReturnType<Service['delete']> {
-    return Promise.resolve(true);
+  public async delete(
+    id: number,
+    user: UserAuthResponseDto,
+  ): ReturnType<Service['delete']> {
+    // const journal = await this.find(); //TODO find(id)
+    const journal = { id: 1, userId: 32 };
```

Comment:
remove this, we have already added user id to journal_entries

Reply:
@Liza-Veis  sry. i forgot to push commit with this changes